packages/loopring_v3/contracts/impl/UserStakingPool.sol

```diff
@@ -195,10 +195,7 @@ contract UserStakingPool is Claimable, ReentrancyGuard, IUserStakingPool
 
         staking.claimedAt = (staking.claimedAt == 0) ?
             staking.depositedAt :
-            staking.balance
-                .mul(staking.claimedAt)
-                .add(additionalBalance.mul(now)) / balance;
-
+            staking.balance .mul(staking.claimedAt) .add(additionalBalance.mul(now)) / balance;
```

Comment:
Strange formatting with empty space before the dot.

Reply:
restored